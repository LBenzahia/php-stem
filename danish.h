
/* This file was generated automatically by the Snowball to ANSI C compiler */

extern struct SN_env * danish_create_env(void);
extern void danish_close_env(struct SN_env * z);

extern int danish_stem(struct SN_env * z);

