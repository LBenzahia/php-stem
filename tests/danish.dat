a
aab
aabenbaringen
ab
abaddon
abagta
aban
abana
abarim
abarimbjergene
abarimbjerget
abba
abda
abdas
abdeels
abdi
abdiel
abdis
abdon
abed
abel
abels
aber
abi
abia
abias
abiasaf
abiathar
abib
abiba
abibajils
abibu
abida
abidan
abidans
abiel
abiezer
abiezers
abiezriten
abiezriterne
abiezriternes
abigajil
abih
abihajil
abihajils
abihu
abija
abijas
abilene
abimael
abiman
abimelek
abimeleks
abimelelk
abinadab
abinadabs
abiner
abinoams
abiram
abirams
abisj
abisjag
abisjaj
abisjajog
abisjua
abisjur
abisjurs
abital
abitub
abitubs
abiud
abne
abner
abners
abrabams
abraham
abrahams
abram
abrams
abrek
abrimelek
abrona
abrÃ¦ndes
absalom
absaloms
absalon
absalon