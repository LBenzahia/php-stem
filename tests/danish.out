a
aab
aabenbaring
ab
abaddon
abagta
aban
abana
abarim
abarimbjerg
abarimbjerg
abba
abda
abda
abdeel
abdi
abdiel
abdis
abdon
abed
abel
abel
aber
abi
abia
abia
abiasaf
abiathar
abib
abiba
abibajil
abibu
abida
abidan
abidan
abiel
abiez
abiez
abiezrit
abiezrit
abiezrit
abigajil
abih
abihajil
abihajil
abihu
abija
abija
abil
abimael
abiman
abimelek
abimelek
abimelelk
abinadab
abinadab
abin
abinoam
abiram
abiram
abisj
abisjag
abisjaj
abisjajog
abisjua
abisjur
abisjur
abital
abitub
abitub
abiud
abn
abn
abn
abrabam
abraham
abraham
abram
abram
abrek
abrimelek
abrona
abr√¶nd
absalom
absalom
absalon
absalon
absjaj
ad
ada
adad
adaj
adaja
adaja
adalja
adam
adama
adama
adami
adam
adan
adan
adar
ada
adb
addan
addar
addis
addon
adel
ade
adf√¶rd
adgang
adgang
adiel
adiel
adin
adina
adin
aditajim
adja
adlaj
adlyd
adlyd
adlyd
adlyd
adl√∏d
adl√∏d
adma
adma
admata
adna
adonibezek
adonija
adonija
adonikam
adoniram
adonizedek
adorajim
adrammelek
adramyttisk
adriatisk
adriel
adskil
adskilt
adsplit
adsplit
adsplitted
adsplitted
adsplit
adsplit
adspred
adspred
adspred
adspred
adspred
adspred
adspred
adspurg
adspurg
adsp√∏rg
adsp√∏rg
adullam
adullam
adummimpas
advar
advared
advar
advar
advar
advarsel
aetaxerx
aezin
af
afaldskorn
afaresit
afarsatkit
afarsekit
afbarked
afbilded
afbild
afbildning
afbildning
afbilled
afbrud
afbr√¶nd
afbr√¶nd
afdeling
afdeling
afd√¶k
afd√∏d
afd√∏d
afd√∏d
afefterkom
afek
afeka
affald
affat
affat
affatted
affatted
affat
aff
affen
affolk
affolked
aff√¶ld
aff√∏r
aff√∏r
aff√∏r
aff√∏rt
aff√∏rt
afgav
afgav
afgift
afgift
afgift
afgift
afgiv
afgiv
afgjord
afgjorift
arift
atajipatajipjipadtkit
orug
afdorug
afdorug
afdor√¶safdor√∏dafdor√∏dafdor√∏dafdor√∏djipjipar√∏djipjipar√∏djipjipar√∏djipfbdatkit
or√∏djipfrit
or√∏djiprit
or√∏djiprit
or√∏dit
or√∏dit
oudit
oudit
oudit
oudit
oudsrud
afbr√oudsrud
afbr√oudsrud
afbr√oudsrud
afbr√oudsrr√∏dit
oudsdyrkit
oudsdyrkit
oudsdyrkit
oudsdyrkit
oudsdyrkit
oudshusit
oudsh√∏jit
oudsipfbdk√∏dit
oudsipfbdk√∏dit
oudsiprit
oudspr√¶stit
oudspr√¶stit
oudsvddisfgt
aff√gt
aff√gt
aff√gt
aff√hjalpff√hjr
apff√hoaff√∏hoaff√∏hoaff√∏hoafaffolhoafaffolhoaffolhoaffolhuggeffolhuggeffolhugfolhugfolhugfolhugffolhinggfol
abifikit
jiblepas
fkaffat
klipat
klipat
kl√¶dat
kl√¶dat
keka
akeka
akeka
akeraff
afkeraff
afkera
afkera
afkeraled
afbkro
afbkr√¶fgiv
kr√¶fgiv
kr√¶viv
kr√¶vff
afkr√¶viv
l
abi
l
abi
levbi
levbi
livbi
livbi
l√¶gbi
l√¶gbi
l√¶gbi
l√¶gbi
l√¶gbi
l√∏sled
afbm
abi
mff
afm√•l
afm√•lt
afm√¶gbi
m√¶gbi
m√¶rkit
m√¶rkff
afm√¶rkit
oaffolr
abi
r
abi
r
abi
rejsbi
revffolrivbi
rivbi
rystit
s
abi
sen
afsejl
afsejlff
afsif
afs
afs
afsind
afsind
afsind
afskorn
askorn
askorfed
afskor
afskor
afsked
afskrafgiv
skrafgiv
skrr
alff
afskuf
afsky
afskyef
afsky
afsky
afskyr
afsk√•r
afsk√¶r
afsk√¶r
afsk√¶r
afslo
afbslugiv
sl√•iv
sl√•iv
sl√∏ref
afslt
aff√slt
aff√sp√¶r
afsp√¶rref
afsp√¶r
afstand
afstand
afstod
afstrorn
ast√•iv
svff
afsvff
afsvif
afs√¶t
afs√¶t
af
abitf
abitf
abtitf
abtitf
itf
enipar√∏djipjip
enipar√∏djiprjip
eneling
iprjip
eneling
iprjip
enjip
enjip
enipjip
enipjip
enskygjip
ensm√•ltif
af
ensm√•ltif
af
enstif
af
enstjem
af
enstug
afd
entif
af
off
aftved
afbtved
afbtvud
afbtv√¶t
af
v√¶t
af
t
aff√
t
aref
af
t
aff√udsrud
afbr√vddtbr√vddtbr√vegbr√vegnbr√vejbr√vejafbr√vejbr√vbr√vdel
√vdel
√vdel
√vdel
√vdekl
√vdekl
√vdetbr√v√¶bnbr√v√¶
aduf√¶datadugabusitgtadugageft
agagd
afgen
aga
abga
¶ndgabga
dyrkitga
dyrkitga
dyrkitga
itga
h√∏mitga
h√∏nitga
rift
aga
rift
aga
rift
aga
land
aga
land
agasitg
itgrippabigtbigtbigtafbrgtafbrgtbrgtbrgtbrgtbrgabitgabkitgabkitgabkhtkitgabklignbr
abbibhx
aehasjtabbtem
ahasverusithasverusithavaithazithazbdlydhazjaydhazjaydha
ithiithian
ahit
abhit
abhihuf
aham
adham
adhaam
adhaam
adhaamadhalbhihufbhihufbtvud
hihufbvud
hg